cartridge/failover.lua

```diff
@@ -418,6 +560,21 @@ local function cfg(clusterwide_config)
     end
 
     accept_appointments(first_appointments)
+
+    local ok, err = constitute_oneself(vars.cache.active_leaders, {
+        timeout = vars.options.WAITLSN_TIMEOUT
+    })
+    if ok == nil then
+        log.warn("Error reaching consistency: %s", err)
+        if next(vars.schedule) == nil then
+            local task = fiber.new(reconfigure_all, vars.cache.active_leaders)
+            local id = task:id()
+            task:name('cartridge.failover.task')
+            vars.schedule[id] = task
+            log.info('Consistency not reached, another attempt scheduled (fiber %d)', id)
```

Comment:
I'm not sure that fiber.id is useful here. Correct me if it's not true.

Reply:
I've added it for debugging and i'd like it to stay. This is the moste consise way to investigate tricky fiber races in logs:

```text
main/164/cartridge.stateful-failover I> Failover triggered, reapply scheduled (fiber 165)
main/165/cartridge.failover.task I> ...
```

Reply:
Ok